gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -2341,9 +2444,7 @@ public String getStudentNumber(final User u, final Site site)
 	 *
 	 * @return
 	 */
-	public Map<String, Set<Member>> getGroupMembers() {
-
-		final String siteId = getCurrentSiteId();
+	public Map<String, Set<Member>> getGroupMembers(final String siteId) {
```

Comment:
In general I don't like the idea of adding the siteId as a param to public methods unless necessary, otherwise this service could be abused by other tools or services. In fact this method could be made private as I believe it's only used by the business service (and in which case the param could stay).